package.json

```diff
@@ -4,8 +4,26 @@
   "description": "Full-stack document management system",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node ./bin/www",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon  --exec babel-node ./bin/www",
+    "test": "npm run setup:test && NODE_ENV=test nyc mocha server/tests/**/*.js client/test/testSetup.js client/test/**/* --timeout 20000",
```

Comment:
I don't know what gets executed in client/test/testSetup.js but I feel since you have a custom "setup:test" command you should move the testSetup script there too.